projects/packages/assets/src/class-script-data.php

```diff
@@ -215,6 +215,10 @@ protected static function get_current_user_data() {
 		return array(
 			'display_name' => $current_user->display_name,
 			'id'           => $current_user->ID,
+			'permissions'  => array(
+				'manage_options' => current_user_can( 'manage_options' ),
+				'manage_modules' => current_user_can( 'jetpack_manage_modules' ),
+			),
```

Comment:
I think these can be added via the connection initial state. It's probably something that we should look at it soon, and have the connection use script-data too.